packages/zoe/src/state.js

```diff
@@ -8,7 +8,9 @@ import { makeTable, makeValidateProperties } from './table';
 // Installation Table
 // Columns: handle | installation
```

Comment:
Looks like you need to add `code` to this comment as an additional comment.

Reply:
done